blocks/columns/columns.css

```diff
@@ -285,15 +340,32 @@
         gap: 80px;
         }
 
+    .content-band div:empty {
+        display: unset;
+        }
+
     .terms > div > div:nth-of-type(even) {
-        flex: 2;
-        margin-left: var(--spacer-layout-04);
+        flex: 1;
+        margin-left: unset;
     }
 
+    .terms > div > div:nth-of-type(odd) {
+        width: 167px;
+        flex: unset;
+        }
+
     .columns > div > div:first-of-type {
         margin-left: unset;
     }
 
+    .teaser-list-manual picture, .teaser-list-manual p > picture {
+        display: unset;
+        }
+
+    .section:not(.leadspace) .image-end picture, .section:not(.leadspace) .image-start picture {
+        display: unset;
+        }
```

Comment:
Could this be combined with the existing style?

```
    .news .columns.quarter-circle picture {
        display: unset;
    }
```

Would

```
    .columns.quarter-circle picture {
        display: unset;
    }
```

satisfy all use cases outside of leadspace?

Reply:
Yes, I deleted the redundant code now, thanks.